app/admin/views.py

```diff
@@ -221,6 +221,47 @@ def destinations_show(uuid):
     )
 
 
+@admin_bp.route('/admin/destinations/<uuid>/modify', methods=['GET', 'POST'])
```

Comment:
I'd rather see us re-use the show destination endpoint + template to process the modification rather than send them to a new endpoint with the form in it.

Reply:
@iandees not sure how to preserve the state of the form between requests

Reply:
What I meant was to show the form (instead of just listing the details) in the `destinations_show()` view and have it also handle the modification of the destination.

Reply:
@iandees without a confirmation step?

Reply:
Yea, for now no confirmation. If that doesn't feel right we could (a) use a JavaScript popup (which we need to figure out how to do eventually) or (b) use a checkbox that confirms the user saw the message.